cmd/argo/commands/common.go

```diff
@@ -21,6 +21,7 @@ var (
 	restConfig       *rest.Config
 	clientConfig     clientcmd.ClientConfig
 	clientset        *kubernetes.Clientset
+	wfClientset      *wfclientset.Clientset
```

Comment:
Can you add a comment for `wfClientset `, to show that this is only for `dry-run` now ?

Reply:
Done